### ISSUE TITLE
TieredMenu: Navigation via keyboard not working

### ISSUE THREAD

**Issue by fcjm** [primefaces/primefaces#6950]
**Describe the defect**
We use the Overlay TieredMenu in our application and found that the keyboard navigation for it is currently not working. This also applies for the Inline TieredMenu. Overlay can be opened via Tab+Enter but you cannot access the lower tiered submenus.

**Reproducer**
See showcase for TieredMenu

**Expected behavior**
A similar functionality to Menu (submenu-navigation via Arrow Keys) -> see Showcase for Menu

**Comment by tandraschko** [comment]
new feature as never implemented before

**Comment by melloware** [comment]
I don't think its just TieredMenu either I think its anything with a SubMenu and I think its also related to this ticket: https://github.com/primefaces/primefaces/issues/6561

I think whole whole Keyboard Trap and focus for menus needs to be redone to align with 508 accessibility standards.

**Comment by Groell-Philipp** [comment]
Comment moved to #6561.

**Comment by melloware** [comment]
Yes your comments could belong under #6561

**Comment by melloware** [comment]
Closing as duplicate of #6561
